Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,36 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+/**  function read code coverage report as DOM */
+function readCodeCoverageReportAsDom(): jsdom.JSDOM {
+    const htmlString: string = fs.readFileSync(path.join(reportDirectory, 'index.html'), 'utf-8');
+    return new JSDOM(htmlString);
+}
+
+/**   function write DOM as html */
+function writeDomAsHtml(dom: jsdom.JSDOM): void {
+    fs.writeFileSync(path.join(reportDirectory, 'index.html'), dom.serialize())
+}
+
+/**   function replace images sources to base64 code in Code Coverage report html */
+function replaceImageSourceToBase64(): void {
+    const imageSizeLimit = 1024;
```

Comment:
What is the unit here (bytes, KB, MB)? Can we make it clear by variable name?

Reply:
Added Kb.